pkg/omf/omf.fish

```diff
@@ -119,13 +123,18 @@ function omf -d "Oh My Fish"
 
     case "t" "theme"
       if test (count $argv) -eq 1
-        set -l ostype (uname)
         set -l theme (cat $OMF_CONFIG/theme)
-        set -l regex "[[:<:]]($theme)[[:>:]]"
-        test "$ostype" != "Darwin"; and set regex "\b($theme)\b"
-
-        omf.packages.list --database --theme | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
-        omf::off
+        set -l highlight_current "(^|\s)($theme)(\s|\$)"
```

Comment:
Does changing the order of the `$` and `\s` help anything? I know it probably shouldn't... Just hoping.
My problem with this regex is that there is no explicit limit on package names (except for the prefixes for repositories). I guess no dev would try to add spaces to a package name, but what if I wanted to be hip and call my theme "w~a~v~e" (or anything like that). This regex will stop highlighting after the 'w'.
Maybe it's for the better to limit package naming possibilities, but I wouldn't allow it in a script before it's documented.